libyul/ObjectParser.cpp

```diff
@@ -100,7 +100,10 @@ shared_ptr<Object> ObjectParser::parseObject(Object* _containingObject)
 			fatalParserError(8143_error, "Expected keyword \"data\" or \"object\" or \"}\".");
 	}
 	if (_containingObject)
+	{
+		ret->subId = _containingObject->subObjects.size();
```

Comment:
Currently, this is only set in the EVM object compiler. I think it is a reference to something from Assembly.cpp, so I don't think we should set it to an arbitrary value here.

Reply:
Got it, I've removed this line and the use of `pathToSubObject`. The value I had set was required for `pathToSubObject` to work.